### ISSUE TITLE
Simplify target 7 equivalence condition

### ISSUE THREAD

**Issue by jim-carciofini** [GaloisInc/pate#415]
With the latest Pate changes the form of the equivalence condition changed a bit and the simplifications that previously applied need to be extended. We are now getting:
```
notp(intle(18, sbvToInteger(readLE4(#x20002d14))))
```
We want:
```
18 > readLE4(#x20002d14)
```

**Comment by jim-carciofini** [comment]
@ben, does eq cond look good now? If yes, please close this issue.

**Comment by jim-carciofini** [comment]
@thebendavis , does this eq cond look good? If yes, please close the ticket.

**Comment by thebendavis** [comment]
Looks great, thanks!

Fixed in 44cae29c17ce35666588b44cbf8a5e0fbe1f9df5
